autorest/azure/auth/auth.go

```diff
@@ -167,6 +168,35 @@ func NewAuthorizerFromFileWithResource(resource string) (autorest.Authorizer, er
 	return autorest.NewBearerAuthorizer(spToken), nil
 }
 
+// NewAuthorizerFromCli creates an Authorizer configured from Azure CLI 2.0 for local development scenarios.
+func NewAuthorizerFromCli() (autorest.Authorizer, error) {
```

Comment:
Nit, `Cli` should be `CLI` (comment applies to all instances.).

Reply:
Done!